app/src/lib/stores/app-store.ts

```diff
@@ -2486,6 +2487,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
     const status = await this._loadStatus(repository)
     if (!status) {
       await this._updateRepositoryMissing(repository, true)
+      this.updateSidebarIndicator(repository, null)
```

Comment:
i think we can call this right after `_loadStatus` with its result regardless of what it returns and then we can remove the `this.updateSidebarIndicator(repository, status)` at the end of this function entirely.

since `updateSidebarIndicator` is async and fairly straightforward i don't think there's risk there.